lib/checks/label/duplicate-img-label.js

```diff
@@ -1,4 +1,4 @@
-const text = axe.commons.text.visible(virtualNode, true).toLowerCase();
+const text = axe.commons.text.visibleVirtual(virtualNode, true).toLowerCase();
```

Comment:
Didn't @dylanb suggest doing this the other way around? (i.e. calling `visibleVirtual` from `visible` so the checks stay the same)